### PR TITLE
SEO-fixes applied according to the audit results

### PR DESCRIPTION
**Pull request by keriat**
1. Duplicated Meta

1.1 Description

The following table lists all pages with identical meta descriptions. Meta description: zkSync is a fully trustless user-centric zkRollup protocol for scaling payments and smart contracts on Ethereum. 

1.2. Duplicate page titles

Different pages with the same title must absolutely be avoided, each page should have its own, individual page title. If you use the same page title for different pages, your page title either lacks expression or it doesn’t match the content of all pages: you give away optimization potential as Google usually only shows one page per domain in the search results. 

In case of duplicate titles, several pages must have been optimized for the same keywords and might therefore have no chance of being found for other, differing keywords. Read Matt Cutts's explains how to create page titles, Google Title Update: Everything You Need To Know. 

1.3 Problematic page titles

✅ Please consider the following tips when choosing a page title:

Length of 55 to 65 characters

Important page keywords should be used in the title

2 to 3 keywords should be logically connected

The most important keywords should appear first in the title

A page’s title should not change too often

Each title should only appear once on the entire website

❌ Common errors when choosing the page title:

Title tag is missing

Title tag is empty

Page title is too long

Page title is too short

Page title is a single word

Page title contains unnecessary repetitions

All subpages use the same page title

The website’s URL is in the title tag

The page title lacks important keywords

The page title contains words such as “welcome” or the domain name

The most important keyword doesn’t appear first in the title tag

Use Free SERP Snippet Generator by Seobility

2. Pages with a big distance to the homepage

Important pages should only be a few clicks away from your homepage. When pages are located very deep in the structure of your site, they will be found difficulty by visitors and search engine bots as well. In addition, these sites are treated as less important by search engines. Make sure that important pages are not located too deep in your site structure. You can link to around 200-300 other pages per page. With a good internal and reciprocal linking of your subpages you avoid deep page levels.

3. Problems with ext. links

External links to other websites are generally favorable and consistent with the concept of interconnection in the world wide web. If you complement your texts with external links to websites with relevant topics, you can create additional value for your visitors. Search engines recognize these links as a sign of quality of good content. After all, in the world of science there aren’t any studies or papers without references or citations either.

Apart from links to other content relevant to your topic, you can also link to websites, suppliers and profiles on social networks that you are in contact with. In case of all external links you should be aware, however, that the content linked may change, usually without giving you notice. Websites may be abandoned, restructured or redirected to other sites, so that your link might eventually no longer lead to the linked site, but to an error page, to other content or even to pages with malware etc.

The following table lists all external linkings from your website that are potentially problematic:

3.1 Excluded pages (?)

Page titles: zkSync: secure, scalable crypto payments

  IMO the pages below should be excluded from the parsing

1.

https://zksync.io/maintenance.html

2.

https://zksync.io/service/down-night.html

3.

https://zksync.io/service/down.html

4.

https://zksync.io/service/testnet-down-night.html

5.

https://zksync.io/service/testnet-down.html [details]

3.2 404 referred page

https://zksync.io/dev/providers.html

404

HTTP status

3.3 Identical anchor text for different pages

Anchor texts are a great way to inform visitors and search engines about the content of the linked page. Use good anchor texts, which contain important keywords for the linked page.

If different pages are linked with the same text or keyword, the reference is not specific enough. So the optimization potential of internal linking remain unused. Trivial link texts like "show more" or "klick" should generally be avoided.

Note: If the page title of a page is not "good enough", Google uses anchor text, which refers to the page, as the title in search result pages.

3.4 Internal anchor texts that need improvement

Make sure that your pages have a good internal linking structure and that your internal links have useful anchor texts. It's recommended to link pages with relevant topics to the keywords that are important for the page as well. But avoid constant repetitions of anchor texts for the very same page, too long anchor texts (max 120 characters) or even links without anchor texts (e.g. in links via pictures or JavaScript). Matt Cutts on the topic of "Too many internal links with the same link text"

4. Content that appears on several pages

Each page should be unique and avoid using paragraphs or similar elements several times. For this analysis, every paragraph found on your pages will be normalized and saved. The analysis will detect identical paragraphs appearing on several pages.

You should absolutely avoid using the same content on several pages. If you use too long paragraphs from blog posts or your blog homepage, for instance, this might negatively impact the ranking for the blog post itself as the search engine will allocate the content only to a specific page of your website to avoid duplicate hits on the same website. So, if you have several pages with the same content, the search results might only list exactly the page that hasn’t been optimized for the searched content and the page will consequently rank lower in the search results.

Comment:Tooltips, disclaimers etc. that appear on a high number of your pages will be disregarded.

5. Problems with H1 headings

Headings give structure to running text and improve its readability for visitors. In addition, headings, above all the H1 heading, give an overview of the page content, similar to the page title. Search engines analyze headings accordingly and regard keywords used in headings as correspondingly important.

In general, headings are hierarchically organized from H1 to H7, the H1 heading being the most important of them. This heading should summarize the complete content of the page with the most important keywords. A page should always contain only one H1 heading. But you can use several H1 headings if the document requires this.

However, there is no reason why you shouldn’t start with the first level H1 when using headings.

✅ Please keep the following in mind when choosing the H1 heading:

There should only be one H1 heading per page

The headings hierarchy should appear in descending order without gaps (H1 is followed by H2 and so on)

The H1 heading should fit to the page content

The H1 tags should contain relevant keywords

The text in the H1 tag should not exceed a length of about 120 characters

Headings should only be used to emphasize the content, not for navigation elements, links etc.

❌ Common mistakes when choosing the H1 heading:

The H1 tag is missing

No content in the H1 tag

The H1 content consists of a single word

The H1 content is too short (e. g. only two very short words)

The H1 tag content is too long (If the H1 tag contains more than 120 characters it might be regarded as keyword stuffing)

The H1 tag content is too general (e. g. welcome)

There are navigation elements and links in the H1 tag

### CODE REVIEW

**Review by bxpana — DISMISSED** [review]

File: docs/dev/contracts/README.md [hunk 1]
```diff
@@ -46,7 +46,11 @@ You can interact completely with smart contracts and the zkSync network via our
 - For _read_ requests: any web3-compliant framework in any language will work out of the box, with additional optional zkSync L2 specific functionality.
 - For _write_ requests (sending transactions): due to fundamental differences between L1 and L2, you will have to write some additional code (for example, zkSync supports paying fees in any token, so sending a transaction will involve choosing a token to pay fees).
 
+<<<<<<< 11/21-SEO-fixes
+You can ask questions and get assistance in our [Discord](https://discord.gg/nMaPGrDDwk) developer chat room.
+=======
```
Comment (StanislavBreadless) [inline docs/dev/contracts/README.md]: merge conflict. please resolve

**Review by StanislavBreadless — APPROVED** [review]
If all the links are tested, then all good :+1:

**Comment by keriat** [comment]
Closes ZKF-1167

**Pull request merged**